app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -551,10 +562,11 @@ public void afterTextChanged(Editable s) {
 
         // These can only be added after everything affected by the media queue is initialized.
         if (!ListUtils.isEmpty(loadedDraftMediaUris)) {
-            for (String uriString : loadedDraftMediaUris) {
-                Uri uri = Uri.parse(uriString);
+            for (int mediaIndex = 0; mediaIndex < loadedDraftMediaUris.size(); ++mediaIndex) {
+                Uri uri = Uri.parse(loadedDraftMediaUris.get(mediaIndex));
                 long mediaSize = getMediaSize(getContentResolver(), uri);
-                pickMedia(uri, mediaSize);
+                String description = loadedDraftMediaDescriptions == null ? null : loadedDraftMediaDescriptions.get(mediaIndex);
```

Comment:
Just had a IndexOutOfBoundsException here after trying to open a toot that was saved on master. I guess a range check would fix this.